commands/serve/docs/fixture-format.md

```diff
@@ -0,0 +1,132 @@
+# Fixture format
+
+## Purpose
+
+The fixture is a set of configurations which specifies the visualization to render and how it behaves. For example, type of visualization, what data to display, theme to use and which feature flags to activate.
+
+Rendering a visualization from a fixture multiple times yields the same result. This makes fixture rendering ideal to use in tests to verify that the visualization works as expected.
+
+## Example
+
+```js
+import gridChart from '@nebula.js/sn-grid-chart';
+
+export default () => {
+  return {
+    type: 'sn-grid-chart',
+    load: async () => gridChart,
+    instanceConfig: {
+      context: {
+        theme: 'dark',
+      }
+    },
+    snConfig: {
+      ...
+    },
+    genericObjects: [ ... ]
+  };
+}
+```
+
+## Fixture configurations
+
+### `type`
+
+Name of visualization. For example, `sn-grid-chart`.
+
+### `load`
+
+Function loading visualization to render.
+
+Example using imports:
+
+```js
+import simpleVisualization from './simpleVisualization';
+export default () => ({
+  load: async () => gridChart,
+  ...
+});
+
```

Comment:
Something is a little wrong with this code snippet.. Don't thing the code block is properly closed here.